### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by Jacky-na**

### CODE REVIEW

**Comment by Jacky-na** [comment]
add a semantic commit or PR title

**Comment by Jacky-na** [comment]
a semantic commit

**Comment by Jacky-na** [comment]
a semantic commit or PR title

**Comment by Jacky-na** [comment]
add a semantic commit or PR title

**Comment by Jacky-na** [comment]
add a semantic commit

**Comment by Jacky-na** [comment]
leava a contact

**Comment by Jacky-na** [comment]
leava a comment

**Comment by Jacky-na** [comment]
ready to be merged or rebased

**Comment by Jacky-na** [comment]
add diff to the end of URLs for Git’s plaintext views.

**Comment by Jacky-na** [comment]
add.diff to the end of URLs for Git’s plaintext views.

**Comment by Jacky-na** [comment]
Attach files by seleting or pasting them.

**Comment by Jacky-na** [comment]
files by seleting

**Comment by dmorais92** [comment]
Dude, you could've just taken a look at the other PR's...
Change the PR title to "feat: updating contributing.md" and you should be good :+1:

File: contributing.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 :+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
 
-This project adheres to the Contributor Covenant [code of conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md).
+This project adheres to code of conduct(https://github.com/electron/CODE OF CONDUC Tmd)
```
Comment (dmorais92) [inline contributing.md]: Typo

File: contributing.md [hunk 2]
```diff
@@ -12,20 +12,20 @@ propose changes to this document in a pull request.
...
-  + [Screenshots](#screenshots)
-  + [Colors](#colors)
-  + [Icons](#icons)
-  + [Locales](#locales)
-  + [Company Logos and Names](#company-logos-and-names)
-  + [Submission Guidelines](#submission-guidelines)
-* [Removing or Disabling Apps](#removing-or-disabling-apps)
-* [Development](#development)
-* [Testing](#testing)
+* [Adding your app]
```
Comment (dmorais92) [inline contributing.md]: Why are you removing this ?

**Pull request closed without merging**